### DISCUSSION TITLE
How can I create a custom Task ID and push to redis

### DISCUSSION THREAD

**Discussion by DilanJT** [apalis-dev/apalis#414] [Q&A]
```
let task_id = match storage.push(updated_job.clone()).await {
        Ok(task_id) => task_id,
        Err(e) => {
            println!("Error pushing job to storage: {}", e);
            return Err(Box::new(e));
        }
    };
    
```
    Instead of getting a generated task_id, I want to give a custom task_id along with storage.push()

**Comment by geofmureithi** [comment]
This will be resolved by #416 as it introduces methods like `push_request` and `schedule_request`.
See an example here https://github.com/geofmureithi/apalis/pull/416/files#diff-d019ef5f40fdc1d870b236e2170b65460839706241f35d6716359f085f9738bdR30

You can easily construct the `Request` the way you want.
